config/crd/kustomization.yaml

```diff
@@ -14,3 +14,12 @@ patches:
   target:
     kind: CustomResourceDefinition
     name: bgppeers.metallb.io
+- patch: |-
+    - op: add
+      path: "/metadata/annotations/metallb-internal"
```

Comment:
cannot omit `metallb-internal`, possibly due to [kustomize bug](https://github.com/kubernetes-sigs/kustomize/issues/4786#issuecomment-1292356615)